core/toolbox.js

```diff
@@ -175,7 +178,8 @@ Blockly.Toolbox.prototype.init = function() {
     oneBasedIndex: workspace.options.oneBasedIndex,
     horizontalLayout: workspace.horizontalLayout,
     toolboxPosition: workspace.options.toolboxPosition,
-    renderer: workspace.options.renderer
+    renderer: workspace.options.renderer,
+    theme: workspace.options.theme
```

Comment:
Do we need this here? I think the workspace is just going to end up using the parent's themeManager

Reply:
Yep, I got rid of all the others, this one snuck through.